specification/cognitiveservices/data-plane/Content Moderator/v1.0/contentmoderatorImageText.swagger.json

```diff
@@ -0,0 +1,591 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Content Moderator Image Text Client",
+    "version": "1.0",
+    "description": "You use the API to scan your content as it is generated. Content Moderator then processes your content and sends the results along with relevant information either back to your systems or to the built-in review tool. You can use this information to take decisions e.g. take it down, send to human judge, etc.\r\n\r\nWhen using the API, images need to have a minimum of 128 pixels and a maximum file size of 4MB. \r\nText can be at most 1024 characters long. \r\nIf the content passed to the text API or the image API exceeds the size limits, the API will return an error code that informs about the issue.\r\n\r\nThis API is currently available in:\r\n\r\n* West US - westus.api.cognitive.microsoft.com\r\n* East US 2 - eastus2.api.cognitive.microsoft.com\r\n* West Central US - westcentralus.api.cognitive.microsoft.com\r\n* West Europe - westeurope.api.cognitive.microsoft.com\r\n* Southeast Asia - southeastasia.api.cognitive.microsoft.com"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{azureRegion}.api.cognitive.microsoft.com",
+    "parameters": [
+      {
+        "$ref": "#/parameters/AzureRegion"
+      }
+    ]
+  },
+  "basePath": "/contentmoderator/moderate/v1.0",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/ProcessImage/FindFaces": {
+      "post": {
+        "tags": [
+          "Images"
+        ],
+        "operationId": "Images_FindFaces",
+        "description": "Returns the list of found faces.",
+        "parameters": [
+          {
+            "name": "CacheImage",
+            "in": "query",
+            "description": "Whether to retain the submitted image for future use; defaults to false if omitted",
+            "type": "boolean"
+          },
+          {
+            "$ref": "#/parameters/ocpApimSubscriptionKeyParameter"
+          },
+          {
+            "$ref": "#/parameters/contentTypeHeader2"
+          },
+          {
+            "name": "file",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "file"
```

Comment:
This seems to be non-confimant to the swagger 2.0 schema. Please use 

```json5
"schema" : {
  "type": "object",
  "format": "file"
}
```

Reference: https://github.com/Azure/azure-rest-api-specs/pull/1544